pkg/apis/kops/cluster.go

```diff
@@ -276,6 +276,9 @@ type KopeioAuthenticationSpec struct {
 }
 
 type AwsAuthenticationSpec struct {
+	// The container image name to use, which by default is:
+	// gcr.io/heptio-images/authenticator:v0.3.0
+	ImageName string `json:"imageName,omitempty"`
```

Comment:
Can we change `ImageName` to be just `Image`, that way we conform with the existing schemas where you can specify the image to be used.
Example: https://github.com/kubernetes/kops/blob/74c880f12edff3ac671a2b45d17cefd3994bfd8c/pkg/apis/kops/cluster.go#L423.

Reply:
I was using #5119 as a reference, but using just `Image` makes sense to me. Updated.